modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/containers/setup-forms/InviteTeamMembersForm/index.js

```diff
@@ -181,7 +182,14 @@ const InviteTeamMembersPage = ({
 					}))
 				);
 			}
-
+			else {
+				setAccountRolesOptions((previousAccountRoles) =>
+					previousAccountRoles.map((previousAccountRoles) => ({
+						...previousAccountRoles,
+						disabled: false,
+					}))
+				);
+			}
```

Comment:
Avoid else statment, use return instead.
To do so, return an empty value (just return;) at the end of the first if statment (line 90)
```suggestion
			
				setAccountRolesOptions((previousAccountRoles) =>
					previousAccountRoles.map((previousAccountRoles) => ({
						...previousAccountRoles,
						disabled: false,
					}))
				);
			
```

Reply:
Nice, thanks for the tip!